flowtools/metacyto_glm_clr/metacyto_glm_clr.R

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env Rscript
+######################################################################
+#                  Copyright (c) 2018 Northrop Grumman.
+#                          All rights reserved.
+######################################################################
+#
+# Version 1 - April 2018
+# Author: Cristel Thomas
+#
+#
+
+library(MetaCyto)
+library(data.table)
```

Comment:
We need to add the data.table conda package to the dependencies in the Galaxy requirements sections.

Reply:
Both dyplr and tidyr are already part of the bioconductor-metacyto package.

Reply:
On quick inspection of the R code though, I don't see any use of data.table; I think it is being imported but not used. I would suggest trying this script within the bioconductor-metacyto 1.4.0 package and removing the library(data.table) line and see if it works.